### ISSUE TITLE
Unknown "ContentType" in Xpath

### ISSUE THREAD

**Issue by amesga** [w3c/publ-a11y#524]
https://github.com/w3c/publ-a11y/blob/459dba68159f90e8bb034ad995731fe51e745a96/a11y-meta-display-guide/2.0/techniques/onix-metadata/index.html#L790

There are multiple instances where `ContentType` is referenced in the Xpath statements, but I don't believe `ContentType` is a valid term. Should this be `PrimaryContentType` or `ProductContentType`? Or both, similar to:

https://github.com/w3c/publ-a11y/blob/459dba68159f90e8bb034ad995731fe51e745a96/a11y-meta-display-guide/2.0/techniques/onix-metadata/index.html#L506

**Comment by rickj** [comment]
@gregoriopellegrino @gautierchomel @avneeshsingh just FYI, this is from one of the VitalSource team members.

**Comment by chrisONIX** [comment]
There are multiple instances where ContentType is referenced in the Xpath statements, but I don't believe ContentType is a valid term. 
This is correct - ContentType does not exist on it's own in ONIX 
Should this be PrimaryContentType or ProductContentType? 
Yes tt should be both

**Comment by clapierre** [comment]
fixe
